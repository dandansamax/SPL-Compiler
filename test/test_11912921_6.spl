int mismatch_retn(float i1, float i2)
{
    return 2.0 * i1 * i2;
}

int test_6()
{
    float a;
    float b[20];
    int c[3];
    a[2] = 3.5;
    a = mismatch_retn(a[2], b);
    c[1] = mismatch_retn(b[1], b[2]);
}