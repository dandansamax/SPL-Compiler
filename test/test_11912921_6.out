Error type 8 at Line 3: return value type mismatches the declared type: The function need return int type, but you return float type.
Error type 10 at Line 11: applying indexing operator on non-array type variables: get float.
Error type 10 at Line 12: applying indexing operator on non-array type variables: get float.
Error type 5 at Line 12: unmatching types appear at both sides of the assignment operator (=): LHS is float, but RHS is int.
